### DISCUSSION TITLE
NATS Witness Node

### DISCUSSION THREAD

**Discussion by CallumLangdaleSaab** [nats-io/nats-server#6108] [Q&A] [answered]
Does anyone know of a way to turn a NATS server into a witness node?
The node should not store or process any data itself, but should be involved in the voting process.

**Comment by neilalexander** [comment, marked as answer]
We don't really have the concept of a witness node and each replicated asset has its own elections, but only nodes that an asset is placed on participate in an election.

If you wanted some nodes to participate in the metalayer election (for tracking which streams/consumers are where) but not actually host any streams/consumers themselves, then you could scale up the cluster and then use placement tags to assign streams/consumers only to a subset of them, but this would do nothing to help stream/consumer availability and I'm not really sure what it would give you.

**Comment by ramonberrutti** [comment]
Hi, @CallumLangdaleSaab. I believe you are looking for the tag `!jetstream` on the server side.  Allows the server to be part of the meta leader, but any stream will be placed there.
